Dockerfile

```diff
@@ -41,7 +41,7 @@ FROM base AS registry
 # both. This allows integration-cli tests to cover push/pull with both schema1
 # and schema2 manifests.
 ENV REGISTRY_COMMIT_SCHEMA1 ec87e9b6971d831f0eff752ddb54fb64693e51cd
-ENV REGISTRY_COMMIT 47a064d4195a9b56133891bbb13620c3ac83a827
+ENV REGISTRY_COMMIT 17b3ff188dfd
```

Comment:
this looks to be v2.7.0-rc.0 https://github.com/docker/distribution/commit/17b3ff188dfde7d5b59a94ab99e6a967b3a59563

Could you use v2.7.1 here (pin by commit: 2461543d988979529609e8cb6fca9ca190dc48da, but add a comment what version it is)

(Also curious; was this needed to make the tests pass?)

Reply:
Fixed. I don't remember why this was needed now, sadly.